src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -13,6 +13,7 @@ import { PhysicalIndexToValueMap as IndexToValueMap } from './../../translations
 const privatePool = new WeakMap();
 const COLUMN_SIZE_MAP_NAME = 'autoColumnSize';
 
+/* eslint-disable jsdoc/require-description-complete-sentence */
 /**
  * @plugin AutoColumnSize
```

Comment:
Yep, jsdoc again .. It seems that when the plugin description is wrapped within `eslint-disable/enable` expression it's not generated at all. After adding the `@class AutoColumnSize` tag right after the `@plugin` tag the plugin appears in the docs.

Please review the other plugins.

Reply:
oh, do I recall well we already had this problem?

Reply:
Maybe with the _Comments_ plugin, I assume.